### ISSUE TITLE
Remmina getting down when used ssh & mc

### ISSUE THREAD

**Issue by worldskat** [FreeRDP/Remmina#1075]
Hi. 
* Client (OS name and version): 
ubuntu 14.04 & ubuntu 16.04
* Remmina version (remmina --version): 
remmina - Version 1.2.0-rcgit-15 (git n/a)
* Desktop environment (GNOME, Unity, KDE, ..): 
awesome
* Connecting to (OS and version): 
ubuntu 14.04 & 16.04
* Connecting via (RDP, VNC, ...): 
ssh
=============================
when i use remmina to connect for my servers over ssh and run on remote server Midnight Commander, to search files i'm use ctrl+s. After this combination remmina crush

**Comment by CS-JS** [comment]
Can confirm that on Ubuntu 16.04 KDE, remmina - Version 1.2.0-rcgit-17 (git n/a), when connected to Debian servers. Just opened MC, after STRG+S remmina gets unresponsive, then a dialogue comes up if I want to close it. Unfortunate, but i could not get a BT on this:

```
[Thread 0x7fffbe93b700 (LWP 11043) exited]
[Thread 0x7fffe0af7700 (LWP 7642) exited]
[Thread 0x7fffe971e700 (LWP 7640) exited]
[Thread 0x7fffe9f1f700 (LWP 7639) exited]
[Thread 0x7ffff7f19a40 (LWP 7638) exited]
[Thread 0x7fffcffff700 (LWP 7647) exited]
[Thread 0x7fffca41c700 (LWP 10873) exited]
[Thread 0x7fffc941a700 (LWP 10875) exited]
[Thread 0x7fffc3fff700 (LWP 10877) exited]
[Thread 0x7fffc2ffd700 (LWP 10879) exited]
[Thread 0x7fffc034b700 (LWP 11167) exited]
[Thread 0x7fffc0b4c700 (LWP 10880) exited]
[Thread 0x7fffc37fe700 (LWP 10878) exited]
[Thread 0x7fffc8c19700 (LWP 10876) exited]
[Thread 0x7fffc9c1b700 (LWP 10874) exited]
[Thread 0x7fffcac1d700 (LWP 10872) exited]
[Thread 0x7fffcbb4a700 (LWP 7652) exited]
[Thread 0x7fffcdffb700 (LWP 7651) exited]
[Thread 0x7fffce7fc700 (LWP 7650) exited]
[Thread 0x7fffceffd700 (LWP 7649) exited]
[Thread 0x7fffcf7fe700 (LWP 7648) exited]
[Thread 0x7fffd4e1a700 (LWP 7646) exited]
[Thread 0x7fffd561b700 (LWP 7645) exited]
[Thread 0x7fffd5e1c700 (LWP 7644) exited]

Program terminated with signal SIGKILL, Killed.
The program no longer exists.
(gdb) 
(gdb) bt
No stack.
(gdb) 
```

**Comment by antenore** [comment]
Closed by #1196
